Response from server

Alenapadsekina opened the pull request:
1) Add teams_request.js file to have the list of teams
2) Populate 1st dropdpwn with the list of teams

Commit:
Response from server

1) Add teams_request.js file to have the list of teams
2) Populate 1st dropdpwn with the list of teams

SergeBondar reviewed (changes requested):
1. please replace var with let
2. please have more clear name for js file

Commit:
Variables names changes

1) Change all VARs to LETs
2) Give more ovious names to variables

Commit:
Add employees response

1) Add employees_request.js to populate employees dropdown
2) Add script to html

Commit:
fix

Commit:
enable 2nd dropdown

Commit:
add default empty option

Commit:
updated the table

status responses come from the BE
